README.md

```diff
@@ -968,6 +968,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * [garro95/priority-queue](https://github.com/garro95/priority-queue)[[priority-queue](https://crates.io/crates/priority-queue)] — A priority queue that implements priority changes. [<img src="https://api.travis-ci.org/garro95/priority-queue.svg?branch=master">](https://travis-ci.org/garro95/priority-queue)
 * [mrhooray/kdtree-rs](https://github.com/mrhooray/kdtree-rs) — K-dimensional tree in Rust for fast geospatial indexing and nearest neighbors lookup
 * [Nemo157/roaring-rs](https://github.com/Nemo157/roaring-rs) – Roaring Bitmaps in Rust [<img src="https://api.travis-ci.org/Nemo157/roaring-rs.svg?branch=master">](https://travis-ci.org/Nemo157/roaring-rs)
+* [nvzqz/byte-set-rs](https://github.com/nvzqz/byte-set-rs) [[byte_set](https://crates.io/crates/byte_set)] — Efficient sets of bytes [<img src="https://github.com/nvzqz/byte-set-rs/workflows/CI/badge.svg">](https://github.com/nvzqz/byte-set-rs/actions?query=workflow%3ACI)
```

Comment:
The reason for the build failing is that this Github CI badge doesn't have a branch specified, and we need that to avoid future problems with badges for badges other than whatever the main one is.